doc/source/_toc.yml

```diff
@@ -282,7 +282,7 @@ parts:
               - file: tune/examples/exercises
                 title: "Exercises"
           - file: tune/faq
-          - file: tune/api_docs/overview.rst
+          - file: tune/api/api.rst
```

Comment:
Just checking, are you keeping `Tune Internals` and `Tune Client API` unchanged intentionally?

Reply:
Yep, for Tune Internals, I wasn't sure how to list out all members/methods with their docstrings if I used autosummary. And since that section is not really meant for external users to search for and use as API calls, I decided to leave it fully expanded.

For Tune Client, there was only 1 class, and I think it is fine as is.

Reply:
Ack, though we should probably move `Callback` and `PlacementGroupFactory` out of `Tune Internals`.

I think it would make sense to move `PlacementGroupFactory` directly into the Trainable `Utilities` section, next to `with_resources`.

`Callback` is a bit strange... it's a public `tune` API, which exposes internals about the trials, and is used by `air.RunConfig`... we should probably revisit this down the line.

Thoughts?

Reply:
Moved Callback + PGF to better places.